core/src/from_meta.rs

```diff
@@ -246,6 +246,48 @@ impl<T: syn::parse::Parse, P: syn::parse::Parse> FromMeta for syn::punctuated::P
     }
 }
 
+/// Parsing support for an array, i.e. `example = "[1 + 2, 2 - 2, 3 * 4]"`.
+impl FromMeta for syn::ExprArray {
+    fn from_value(value: &Lit) -> Result<Self> {
+        if let Lit::Str(ref ident) = *value {
+            ident
+                .parse::<syn::ExprArray>()
+                .map_err(|_| Error::unknown_lit_str_value(ident))
+        } else {
+            Err(Error::unexpected_lit_type(value))
+        }
+    }
+}
+
+/// Parsing support for an unsigned integer array, i.e. `example = "[1, 2, 3, 4]"`.
+macro_rules! from_numeric_array {
+    ($ty:ident) => {
+        impl FromMeta for Vec<$ty> {
```

Comment:
```suggestion
macro_rules! from_numeric_array {
    ($ty:ident) => {
        /// Parse an unsigned integer array, i.e. `example = "[1, 2, 3, 4]"`.
        impl FromMeta for Vec<$ty> {
```

Reply:
Unless I'm very much mistaken, putting the comment here will cause it to show up for the emitted impls, which will let someone see the syntax to pass this.

Reply:
No worries, I had thought about that but can't recall why I chose to document the macro over the impl.